src/components/task-details-page/TaskDetailsPage.jsx

```diff
@@ -18,7 +18,7 @@ export default function TaskDetailPage() {
         dispatchTaskList({ type: "TASK_COMPLETE",payload:{id} });
     };
 
-
+    useDocumentTitle(title)
```

Comment:
I think you forgot to pass the actual title here

Reply:
No actually I wanted to display the title based on the task title I want to change the title